apps/admin-x-activitypub/src/views/Explore/Explore.tsx

```
@@ -0,0 +1,12 @@
+import Layout from '@components/layout';
+import React from 'react';
+
+const Profile: React.FC = ({ }) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)